src/sidebar/components/FilterStatus.js

```diff
@@ -70,43 +71,53 @@ function FilterStatusPanel({
 }) {
   const store = useStoreProxy();
   return (
-    <div className="FilterStatus">
-      {store.isLoading() ? (
-        <div className="hyp-u-layout-row--center">
+    <Card classes="mb-3 p-3">
+      <div className="flex items-center justify-center space-x-1">
+        {store.isLoading() ? (
           <Spinner />
-        </div>
-      ) : (
-        <div className="hyp-u-layout-row--align-center">
-          <div className="FilterStatus__text">
-            {resultCount > 0 && <span>Showing </span>}
-            <span className="filter-facet">
-              {resultCount > 0 ? resultCount : 'No'}{' '}
-              {resultCount === 1 ? entitySingular : entityPlural}
-            </span>
-            {filterQuery && (
-              <span>
-                {' '}
-                for{' '}
-                <span className="filter-facet--pre">{`'${filterQuery}'`}</span>
+        ) : (
+          <>
+            <div
+              className={classNames(
+                // Setting `min-width: 0` here allows wrapping to work as
+                // expected for long `filterQuery` strings. See
+                // https://css-tricks.com/flexbox-truncated-text/
```

Comment:
Ah ... this is a subtle issue with direct text children vs text in child elements. The css-tricks reference is definitely needed here.